resources/js/Pages/Collections/Components/FeaturedCollections/FeaturedCollectionsItem.tsx

```diff
@@ -0,0 +1,158 @@
+import cn from "classnames";
+import React from "react";
+import { useTranslation } from "react-i18next";
+import { ButtonLink } from "@/Components/Buttons/ButtonLink";
+import { GridHeader } from "@/Components/GridHeader";
+import { Heading } from "@/Components/Heading";
+import { Img } from "@/Components/Image";
+import { NetworkIcon } from "@/Components/Networks/NetworkIcon";
+import { CollectionNft } from "@/Pages/Collections/Components/CollectionNft";
+import { FormatCrypto } from "@/Utils/Currency";
+
+const truncateDescription = (
+    description: App.Data.Collections.CollectionFeaturedData["description"],
+): string | null => {
+    if (description !== null && description.length > 120) {
+        return description.slice(0, 120) + "...";
+    }
+
+    return description;
+};
+
+const FeaturedCollectionInfo = ({ data }: { data: App.Data.Collections.CollectionFeaturedData }): JSX.Element => {
+    const { t } = useTranslation();
+
+    const defaultNftCardStyles =
+        "bg-white dark:bg-theme-dark-900 grid w-full h-full min-w-full lg:min-w-fit lg:w-52 lg:h-fit";
+
+    const token: Pick<App.Data.Token.TokenData, "symbol" | "name" | "decimals"> = {
+        name: "",
+        symbol: data.floorPriceCurrency ?? "",
+        decimals: data.floorPriceDecimals ?? 18,
+    };
+    return (
+        <div className="left-0 top-0 z-10 w-full p-6 md-lg:p-8">
+            <div className="flex flex-col gap-6 md-lg:flex-row md-lg:justify-between md-lg:gap-8">
+                <div className="flex flex-col md-lg:max-w-[448px] md-lg:py-2 xl:w-[460px] 2xl:max-w-[600px]">
+                    <div className="flex flex-row gap-4">
+                        <div className="relative h-12 w-12 shrink-0">
+                            <Img
+                                wrapperClassName="aspect-square"
+                                className="h-full w-full rounded-full object-cover"
+                                src={data.image}
+                                isCircle
+                            />
+
+                            <div className="absolute bottom-0 right-0 block h-4 w-4 rounded-full ring-4 ring-theme-secondary-50 dark:ring-theme-dark-950">
+                                <NetworkIcon networkId={data.chainId} />
+                            </div>
```

Comment:
I noticed the popular collections table also uses a component like this, can it be combined so we can reuse it where needed instead
<img width="187" alt="image" src="https://github.com/ArdentHQ/dashbrd/assets/35610748/d8861700-fd19-440c-8f11-45dde857a7c5">